src/cli/cli.cpp

```diff
@@ -1287,6 +1290,51 @@ void Interpreter::ProcessCounters(uint8_t aArgsLength, char *aArgs[])
     AppendResult(error);
 }
 
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
+void Interpreter::ProcessCsl(uint8_t aArgsLength, char *argv[])
+{
+    otError error = OT_ERROR_NONE;
+
+    if (aArgsLength == 0)
+    {
+        mServer->OutputFormat("Channel: %u\r\n", otLinkCslGetChannel(mInstance));
+        mServer->OutputFormat("Period: %u(in units of 10 symbols), %ums\r\n", otLinkCslGetPeriod(mInstance),
+                              otLinkCslGetPeriod(mInstance) * kUsPerTenSymbols / 1000);
+        mServer->OutputFormat("Timeout: %us\r\n", otLinkCslGetTimeout(mInstance));
+    }
+    else if (aArgsLength == 2)
+    {
+        long value;
+
+        SuccessOrExit(error = ParseLong(argv[1], value));
+
+        if (strcmp(argv[0], "channel") == 0)
+        {
+            SuccessOrExit(error = otLinkCslSetChannel(mInstance, static_cast<uint8_t>(value)));
+        }
+        else if (strcmp(argv[0], "period") == 0)
+        {
+            SuccessOrExit(error = otLinkCslSetPeriod(mInstance, static_cast<uint16_t>(value)));
+        }
+        else if (strcmp(argv[0], "timeout") == 0)
+        {
+            SuccessOrExit(error = otLinkCslSetTimeout(mInstance, static_cast<uint32_t>(value)));
+        }
+        else
+        {
+            ExitNow(error = OT_ERROR_INVALID_ARGS);
+        }
+    }
+    else
+    {
+        ExitNow(error = OT_ERROR_INVALID_ARGS);
+    }
```

Comment:
nit: If `error` is initialized as  `OT_ERROR_INVALID_ARGS`, this branch could be removed.